### PR TITLE
Adds the new assessment Component

### PR DESCRIPTION
**Pull request by BenHSmith13**
# Code-review checklist #

  * Documentation
      * [ ] Interfaces describe how other programmers would use them.
      * [ ] Code that is not obvious has documentation describing why
        that code is the way it is.

### CODE REVIEW

File: client/js/_author/components/assessments/assessment_form.jsx [hunk 1]
```diff
@@ -1,46 +1,84 @@
...
-              <div className="c-input__bottom" />
-            </div>
-          </label>
+        <AssessmentItems
+          items={this.props.items}
+          editItem={(itemIndex, field, data) =>
+            this.props.editItem(itemIndex, field, data)}
+          addItem={() => this.props.addItem()}
+        />
+        <div className="c-question-add">
```
Comment (dspencer001) [inline client/js/_author/components/assessments/assessment_form.jsx]: The add button should always go at the bottom

Comment (BenHSmith13) [inline client/js/_author/components/assessments/assessment_form.jsx]: thats not how brandons designs have them

Comment (BenHSmith13) [inline client/js/_author/components/assessments/assessment_form.jsx]: it should probably disappear when the thing is open

**Comment by dspencer001** [comment]
Looks good.

**Pull request merged**